shell/models/fleet.cattle.io.gitrepo.js

```diff
@@ -367,26 +382,25 @@ export default class GitRepo extends SteveModel {
         };
 
         out.push({
-          key:                    `${ r.id }-${ c.id }-${ r.type }-${ r.namespace }-${ r.name }`,
-          tableKey:               `${ r.id }-${ c.id }-${ r.type }-${ r.namespace }-${ r.name }-${ randomStr(8) }`,
-          kind:                   r.kind,
-          apiVersion:             r.apiVersion,
-          type:                   r.type,
```

Comment:
some of these were cut out. little bit nervous of this without tracing through all usages

- type
- id - this should be kept if these are used in a list
- clusterName -lots of hits
- namespacedName - not many hits, might be safe?